assignment4.Rmd

```diff
@@ -116,3 +122,18 @@ biplot(pca)
 
 ## Also in this repository is a data set collected from class that shows how many students thought that a TC program was related to andother TC program. Use PCA to look for components of related programs. Explain why you think there are relationships between these programs.
 
+```{r}
+TCprograms <- read.csv("TCprograms.csv",sep=" ")
+pca3 <- prcomp(TCprograms[,-1], scale = TRUE)
+summary(pca3)
+```
+```{r}
+T1 <- TCprograms[,-1]
+T1 <- T1[,1:9]
+COR3 <- cor(T1)
+corrplot(COR3, order="AOE", method="circle", tl.pos="lt", type="upper",        
+tl.col="black", tl.cex=0.6, tl.srt=45, 
+        addCoef.col="black", addCoefasPercent = TRUE,
+        sig.level=0.50, insig = "blank")
+
```

Comment:
Good work. Could further interpret your results and think about the calculation of the composites for each student.